src/clean.cc

```diff
@@ -180,15 +180,21 @@ int Cleaner::CleanTargets(int target_count, char* targets[]) {
   Reset();
   PrintHeader();
   for (int i = 0; i < target_count; ++i) {
-    const char* target_name = targets[i];
-    Node* target = state_->LookupNode(target_name);
-    if (target) {
-      if (IsVerbose())
-        printf("Target %s\n", target_name);
-      DoCleanTarget(target);
-    } else {
-      Error("unknown target '%s'", target_name);
+    string target_name = targets[i];
+    uint64_t slash_bits;
+    string err;
+    if (!CanonicalizePath(&target_name, &slash_bits, &err)) {
```

Comment:
Can I ask you to show err if it fails to canonicalize?
```
Error("Failed to canonicalize '%s': %s", target_name,c_str(), err.c_str());
```

Reply:
This doesn't look addressed.

Reply:
done